doc/protocol/README.md

```diff
@@ -0,0 +1,18 @@
+# dGrants Protocol
+
+The dGrants protocol specification
+
+## Directory Structure
+
+.
+├── protocol.md                   # dGrants protocol definition
+├── examples                      # Example files for metadata and schemas
```

Comment:
What are the `indexer*` files in this folder? Seems like they're both for the Graph, but I'm not totally sure. I think in general another one of these directory structure blocks that explains all the files in `examples` would be helpful

```suggestion
├── examples                                                # Example files for metadata and schemas
               ├── example-collection.json        # explanation of file
               ├── indexer-manifest.yaml.          # explanation of file
               ├── etc...
```

Reply:
I removed the indexer files as they exist in the graph repo separately and i added descriptions for each of the example files remaining.